php/commands/shell.php

```diff
@@ -35,7 +35,7 @@ public function __invoke( $_, $assoc_args ) {
 		if ( '\\Psy\\Shell' == $class ) {
 			\Psy\Shell::debug();
 		} else {
-			$repl = new $class( 'wp> ' );
+			$repl = new $class(  "\nwp> " );
```

Comment:
@JRGould I noticed this adds a newline when the shell first begins too. Is there a way of preventing this?